### PR TITLE
Improved: show bigmarkasread button only, when there are unread articles

### PR DESCRIPTION
**Pull request by math-GH**
It will improve the UX of the "Mark all as read" button

New:

If there is no unread article, then the "Mark all as read" button is hidden.
grafik

If min. 1 article is marked as unread, then the button is shown.
grafik

The button's visibility toggles when the user toggle the read status of the articles.

Changes proposed in this pull request:

- some little JavaScript
- PHTML: the "There are no more articles" text is not anymore part of the button

How to test the feature manually:

1. load the article list
2. toggle the read/unread status of the articles

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

**Comment by Alkarex** [comment]
Maybe an unwanted side-effect: some people might use this button anyway to move to the next view (next feed / next category...)

**Comment by math-GH** [comment]
> Maybe an unwanted side-effect: some people might use this button anyway to move to the next view (next feed / next category...)

Loud thoughts:
It depends on this config
grafik

So it would mean, that a button "go to next unread feed/category" would be better in that case.

**Comment by math-GH** [comment]
Added:

If there is no unread article, but the option `jump to next unread sibling (feed or category)` is enabled:
grafik

if there is unread article and the option `jump to next unread sibling (feed or category)` is enabled:
grafik

File: p/scripts/main.js [hunk 1]
```diff
@@ -1661,6 +1662,23 @@ function refreshUnreads() {
 	req.send();
 }
 
+function toggle_bigMarkAsRead_button() {
+	var bigMarkAsRead_button = document.getElementById('bigMarkAsRead');
```
Comment (Alkarex) [inline p/scripts/main.js]:
```suggestion
	const bigMarkAsRead_button = document.getElementById('bigMarkAsRead');
```

**Comment by math-GH** [comment]
ref #2016

File: app/views/helpers/stream-footer.phtml [hunk 2]
```diff
@@ -30,19 +30,20 @@ if ($hasAccess) { ?>
...
 			class="as-link <?= FreshRSS_Context::$user_conf->reading_confirm ? 'confirm" disabled="disabled' : '' ?>"
 			form="stream-footer"
 			formaction="<?= Minz_Url::display($url_mark_read) ?>"
 			type="submit">
-			<?= _t('gen.stream.nothing_to_load') ?><br />
 			<span class="bigTick">✓</span><br />
-			<?= _t('gen.stream.mark_all_read') ?>
+			<span class="markAllRead"><?= _t('gen.stream.mark_all_read') ?></span>
+			<?php if (FreshRSS_Context::$user_conf->onread_jump_next) { ?>
+				<div class="jumpNext"><?= _t('conf.reading.jump_next') ?></div>
```
Comment (Alkarex) [inline app/views/helpers/stream-footer.phtml:42]: Element `div` is not allowed as child of element `button`.
Should be replaced by a `span` after checking that styles and scripts are OK

**Pull request merged**